README.md

```diff
@@ -1 +1,24 @@
-# mackerel-plugin-registry
\ No newline at end of file
+# mackerel-plugin-registry
+
+Official mackerel and check plugin registry.  Registered plugins can be installed by `mkr plugin install <plugin_name>`.
```

Comment:
I'm not sure "Official" is proper naming for this.  Registered plugins can be created/maintained by users, not by Mackerel team.